cli/dcoscli/data/help/config.txt

```diff
@@ -11,7 +11,10 @@ Usage:
 
 Commands:
     set
-        Add or set a DC/OS configuration property.
+        Add or set a DC/OS configuration property. The available configuration properties
+        are: core.dcos_acs_token, core.dcos_url, core.ssl_verify, dcos_acs_token, dcos_url,
+        mesos_master_url, pagination, reporting, ssl_verify, and timeout.
```

Comment:
dcos_acs_token and dcos_url are listed twice. Lets add core. to all all the properties, and remove reporting since it does nothing atm.